spec/lib/agris/client/grain/purchase_contracts_spec.rb

```diff
@@ -23,14 +23,24 @@
     context 'when a contract is found' do
       let(:fixture_file) { 'grain/purchase_contract_one_result.xml' }
       let(:contract_location) { 'ABC' }
-      let(:contract_number) { '0000011' }
+      let(:contract_number) { '0000025' }
+
+      let(:remark) { 'Test remark #14' }
 
       it 'returns the contract' do
         result = client.purchase_contract(contract_location, contract_number)
 
         expect(result.documents.length).to eq(1)
         expect(result.documents[0].contract_number).to eq(contract_number)
       end
+
+      it 'returns the contract remarks' do
+        result = client.purchase_contract(contract_location, contract_number)
+
+        expect(result.remarks.length).to eq(1)
+        expect(result.remarks[0]).to have_key('remark')
+        expect(result.remarks[0]['remark']['value']).to eq(remark)
```

Comment:
I might be confused here, but we want to make sure we grab remark 14 right?

Wouldn't this give you remark 12 if 14 was empty?

I don't know what they use different remarks for on Contracts but we definitely want to be sure to always grab 14 and not another since that's the one Patrick said to use.

Reply:
Makes sense, I will fix it.

Reply:
I just added extra remarks in staging for this purpose.